test/core/test_compute.py

```diff
@@ -79,12 +81,44 @@ def my_cube_func(analysed_sst: np.ndarray,
         self.assertEqual(0, len(calls))
         self.assertEqual(('time', 'lat', 'lon'), output_var.dims)
         self.assertEqual((6, 180, 360), output_var.shape)
+        self.assertEqual(((3, 3), (90, 90), (90, 90, 90, 90)), output_var.chunks)
 
         values = output_var.values
         self.assertEqual(2 * 2 * 4, len(calls))
         self.assertEqual((6, 180, 360), values.shape)
-        self.assertEqual(275.3 + 0.5 * 2.1, values[0, 0, 0])
-        self.assertEqual(275.3 + 0.5 * 2.1, values[-1, -1, -1])
+        self.assertAlmostEqual(275.3 + 0.5 * 2.1, values[0, 0, 0])
+        self.assertAlmostEqual(275.3 + 0.5 * 2.1, values[-1, -1, -1])
+
+    def test_from_two_inputs_reduce_time(self):
```

Comment:
For the sake of clarity, it would be appropriate to move this test to a new class ComputeDatasetTest and to run it with a dataset that is not a cube.

Reply:
Later